### ISSUE TITLE
Chromecast Audio Support

### ISSUE THREAD

**Issue by alexmat** [tomahawk-player/tomahawk#369]
Will Tomahawk support Chromecast anytime soon? I'm completely dependent on them :/

**Comment by muesli** [comment]
Yeah, there's hope :-) It depends on the integration of support in VLC3, which is being worked on.

**Comment by lokke-moto** [comment]
I hear that Chromecast support is now in the nightly builds of VLC3, is there a way for me to get that working with the nightly builds of Tomahawk-Player?

**Comment by lokke-moto** [comment]
VLC 3 is now out and it has working chromecast support, does that mean we can expect something similar from Tomahawk?
